replace_containerd_start_arktos_api.sh

```diff
@@ -0,0 +1,13 @@
+cd $HOME
+wget -qO- https://github.com/futurewei-cloud/containerd/releases/download/tenant-cni-args/containerd.zip | zcat > /tmp/containerd
+sudo chmod +x /tmp/containerd
+sudo systemctl stop containerd
+sudo mv /usr/bin/containerd /usr/bin/containerd.bak
+sudo mv /tmp/containerd /usr/bin/
+sudo systemctl restart containerd
+sudo systemctl start docker
```

Comment:
Why is this needed?

Reply:
We need replace arktos containerd with a modified version in order for mizar to work. If we dont do this, arktos does not pass tenant info to mizar

Reply:
What does that statement do if docker is already running? Also, are we now not using containerd instead of docker?